### DISCUSSION TITLE
Prevention policy Names in UI are not populating in API response

### DISCUSSION THREAD

**Discussion by Ni-ka-sH** [CrowdStrike/falconpy#1227] [Q&A] [answered]
Hi Team,

Recently, we got a new API tech alert from CrowdStrike mentioning that we have new field names in the prevention policy API.

![image](https://github.com/user-attachments/assets/809ec5bc-c5e4-4d3c-ae86-d09c71a55938)

The API responses are working fine, but in the response which I get, I don't see any fields showing the name in changes.

Example -

![image](https://github.com/user-attachments/assets/365de2d3-03dd-4fbc-a44b-68e2e4b0f3aa)
 
This is the UI page, where you can see setting names like - 'End User Notifications' , 'Unknown executable analysis', etc...

But in the API response, Im not getting the exact name matching in any of the fields like 'id' or 'name'

they are still the old names...

![image](https://github.com/user-attachments/assets/51eba719-b0cc-4340-afdc-0d374454587b)

API i ussed - "/policy/entities/prevention/v1"

Am i missing something here ?

Thanks in advance.

**Comment by crowdstrikedcs** [comment, marked as answer]
Hi @Ni-ka-sH

Thanks for the question! This is slightly outside the scope of FalconPy functionality as it relates specifically to the data returned by the CrowdStrike API. Since we only return what's provided to us this comes down to product level changes. 

I am taking a look at the Tech Alert in relation to this. From what I can see the scope of this change was related to UI based elements only:
`The functionality of the Prevention Policy settings and associated APIs will not be changed`

Does this discrepency in naming impact the usage of your downstream integrations? 
One thing the Tech Alert does provide is the mapping between `API ID Value` and `Current Name`. This could be used to map a given setting to it's name in the UI based on it's ID. 

Let me know with any questions.
